framework/src/mesh/MooseMesh.C

```diff
@@ -75,6 +75,11 @@ MooseMesh::validParams()
                              "REPLICATED: Always use libMesh::ReplicatedMesh "
                              "DISTRIBUTED: Always use libMesh::DistributedMesh");
 
+  params.addParam<bool>(
+      "use_distributed_mesh",
+      true,
+      "If use_distributed_mesh=true, distributed mesh is adopted in calculation.");
```

Comment:
I'm confused.  How is this intended to differ from the existing `parallel_type=distributed` option?  Where is the new `_use_distributed_mesh` member here used?

Reply:
Oh, maybe I made a stupid mistake (in discussion of #22135), in fact, this addition in unnecessary through adding `parallel_type=distributed` in `[Mesh]`. Now I delete this part.

Thank you very much for your comments.